cla_frontend/settings/base.py

```diff
@@ -179,21 +179,49 @@ def sentry_traces_sampler(sampling_context):
     ),
 )
 
-MIDDLEWARE_CLASSES = (
-    "core.middleware.MaintenanceModeMiddleware",
-    "django_statsd.middleware.GraphiteRequestTimingMiddleware",
-    "django_statsd.middleware.GraphiteMiddleware",
-    "django.contrib.sessions.middleware.SessionMiddleware",
-    "django.middleware.common.CommonMiddleware",
-    "django.middleware.csrf.CsrfViewMiddleware",
-    "django.contrib.auth.middleware.AuthenticationMiddleware",
-    "core.session_security.middleware.SessionSecurityMiddleware",
-    "django.contrib.messages.middleware.MessageMiddleware",
-    "django.middleware.clickjacking.XFrameOptionsMiddleware",
-    "cla_auth.middleware.ZoneMiddleware",
-    "core.middleware.Cla401Middleware",
-    "csp.middleware.CSPMiddleware",
-)
+DISABLE_SAMESITE_MIDDLEWARE = os.environ.get("DISABLE_SAMESITE_MIDDLEWARE", False)
```

Comment:
I think this line should read:
`DISABLE_SAMESITE_MIDDLEWARE = os.environ.get("DISABLE_SAMESITE_MIDDLEWARE", "False") == "True"`

This is because on line 184 you evaluate this value as a boolean. In python all strings, except "" evaluate to `True`.